lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-network_manager-cloud_subnet.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_NetworkManager_CloudSubnet < MiqAeServiceCloudSubnet
+    expose :delete_cloud_subnet_queue
```

Comment:
@Ladas If I understood correctly we want to expose all methods which use Openstack API calls, including the deleting operations for Subnets, Networks, Routers, etc. I should clarify it on gitter.

@gmcculloug yes, the method expects a `userid` to be passed. I assumed that user can get his `userid` from the `User` model. Does it seem correct to you?